src/components/search-results/index.js

```diff
@@ -111,9 +112,9 @@ class SearchResults extends Component {
               <VisbilitySensor key={index} onChange={(isVisible) => this.handleVisibility(isVisible, item)}>
                 <div key={index} className='gridItem'>
                   <div onClick={() => removeTile(item.id)}>
-                    <img className='removeTileButton' src='../../src/assets/cancel.svg' alt='cancel' />
+                    <img className='removeTileButton' src={removeTileButton} alt='cancel' />
                   </div>
-                  <div onClick={() => changeRoute(`/article/${item.url}`)}>
+                  <div onClick={() => changeRoute(`/article/${resultId}/${item.id}`)}>
```

Comment:
using the `url` sent in the package/article means we can change it later and not have to update the UI.
this will still work. however it may conflict with what @Kumjami is trying to debug...